shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -242,6 +246,9 @@ static bool DidFlagChange(const flutter::SemanticsNode& oldNode,
 }
 
 void AccessibilityBridge::clearState() {
+  for (SemanticsObject* object in objects_.get().allValues) {
+    object.platformViewSemanticsContainer = nil;
```

Comment:
We can add this in the `dealloc` of `SemanticsObject`

Reply:
Emm, actually we did it in `dealloc` of `SemanticsObject`, https://github.com/flutter/engine/blob/fb208b486ec354dc7324e4228d9c311208174b18/shell/platform/darwin/ios/framework/Source/SemanticsObject.mm#L198, but cause of retain cycle, dealloc not executed.